### PR TITLE
multiple info blocks not allowed for same language

### PR DESCRIPTION
**Pull request by biplovbhandari**
@flavour Please Merge

### CODE REVIEW

File: controllers/cap.py [hunk 1]
```diff
@@ -867,7 +867,21 @@ def postp(r, output):
                                                   limitby=(0, 1)).first()
             iquery = (itable.alert_id == alert.template_id) & \
                      (itable.deleted != True)
-            irows = db(iquery).select(itable.id)
+            irows = db(iquery).select(itable.id, itable.language)
+            # An alert can contain two info segments, one in English and one in
+            # local language
+            if len(irows) > 2:
+                session.error = T("An alert can contain maximum of two info segments!!! Check your template!!!")
+                redirect(URL(c="cap", f="alert", args=[lastid,"info"]))
```
Comment (flavour) [inline controllers/cap.py]: PEP8 space

File: controllers/cap.py [hunk 2]
```diff
@@ -867,7 +867,21 @@ def postp(r, output):
...
+            else:
+                if len(irows) == 2:
+                    # Check if both info segments are for same language
+                    if irows[0]["language"] == irows[1]["language"]:
+                        session.error = T("Please edit already created info segment with same language!!!")
+                        redirect(URL(c="cap", f="alert", args=[lastid, "info"]))
+                if not all(language in [key for key in settings.get_L10n_languages()] for language in [irow.language for irow in irows]):
+                    session.error = T("An alert cannot contain other than English and Local Language!!! Check your template!!!")
+                    redirect(URL(c="cap", f="alert", args=[lastid,"info"]))
```
Comment (flavour) [inline controllers/cap.py]: PEP8 space after comma

File: controllers/cap.py [hunk 3]
```diff
@@ -867,7 +867,21 @@ def postp(r, output):
                                                   limitby=(0, 1)).first()
             iquery = (itable.alert_id == alert.template_id) & \
                      (itable.deleted != True)
-            irows = db(iquery).select(itable.id)
+            irows = db(iquery).select(itable.id, itable.language)
+            # An alert can contain two info segments, one in English and one in
+            # local language
+            if len(irows) > 2:
+                session.error = T("An alert can contain maximum of two info segments!!! Check your template!!!")
```
Comment (flavour) [inline controllers/cap.py]: Do we really need triple exclamation marks? 1 is plenty sufficient I think?

**Comment by biplovbhandari** [comment]
@flavour please merge

**Pull request merged**